### PR TITLE
sgt-back-end - Ken Kieu

### PR DESCRIPTION
**Pull request by kenkieu**
Completed sgt-back-end challenge.

Screen captures below organized by request method!

### CODE REVIEW

**Comment by kenkieu** [comment]
DELETE
DEL 404
DEL 204
DEL 400
DEL 500

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    });
+  } else if (!newStudent.score) {
+    res.status(400).json({
+      error: '"score" is a required field'
+    });
+  } else {
+    const params = [];
+    for (const key in newStudent) {
+      params.push(newStudent[key]);
+    }
```
Comment (thebearingedge) [inline sgt-back-end/index.js:58]: It's not a good idea for your code to make assumptions about property order. You can populate your params manually.

```js
const params = [newStudent.name, newStudent.course, newStudent.score];
```

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    });
+  } else if (!updateStudent.score) {
+    res.status(400).json({
+      error: '"score" is a required field'
+    });
+  } else {
+    const params = [];
+    for (const key in updateStudent) {
+      params.push(updateStudent[key]);
+    }
```
Comment (thebearingedge) [inline sgt-back-end/index.js:101]: Here too.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great job, just two minor changes.

**Pull request merged**